flink-core/src/main/java/org/apache/flink/configuration/description/TextElement.java

```diff
@@ -53,6 +55,11 @@ public static TextElement text(String text) {
         return new TextElement(text, Collections.emptyList());
     }
 
+    /** Wraps a list of {@link InlineElement}s into a single {@link TextElement}. */
+    public static InlineElement wrap(List<InlineElement> elements) {
```

Comment:
wouldn't be a vararg more handy?

Reply:
btw we should add a test for it, it is not used anywhere in this PR right?

Reply:
It's used in `ConfigOptionsDocGenerator`. There the list signature is also definitely more convenient, but I can change it to varargs.